### DISCUSSION TITLE
non-interactive update?

### DISCUSSION THREAD

**Discussion by tessus** [searxng/searxng#2473] [Q&A] [answered]
How can I update the instance with `searxng.sh instance update` but in a way that I am not prompted for an input. I want `2` to be selected and used automatically.  

```
Menu::
  1.) replace file
  2.) leave file unchanged [default]
  3.) interactive shell
  4.) diff files
choose next step with file /etc/searxng/settings.yml [2]
```

I know I can use `expect` but is there a searxng specfic approach? e.g. an env var or a flag?

**Comment by return42** [comment, marked as answer]
You can set environment [`FORCE_TIMEOUT=0`](https://docs.searxng.org/utils/index.html#force-timeout):

    FORCE_TIMEOUT=0 ./utils/searxng.sh instance update

**Reply by tessus** [reply]
When I read the name it means that the default is picked. However, I want choice number 2 to be picked. It's currently the default so that's fine, bu what if that changes? I rather need something like `FORCE_TIMEOUT=0` and `MENU_ITEM=2`

**Reply by return42** [reply]
The default will not be changed / see PR https://github.com/searxng/searxng/issues/1160 .. Its also the way I and many other update theirs instances.
